lib/js/components/SelectList/SelectListItemDivider/SelectListItemDivider.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+	<div class="selectListItemDivider">
+		<divider />
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/spacings';
+
+.selectListItemDivider {
+	padding: $space-xxs 0;
```

Comment:
in `SelectListItem.vue` you define transparent background using a color token. Why we don't do that here?

Reply:
I didn't touch any styles in this PR. `SelectListItem` changes its background depending on the state, but divider doesn't, maybe that's why. Let me know if there is a good reason to add it here, and I'll do that.

Reply:
I remember UX wanted to have these tokens used in case of transparent background, don't remember why

Reply:
I don't see explanation, why you resolved this comment without fixing. Can you explain?

Reply:
Damn it, I added it in the wrong component (https://github.com/bethinkpl/design-system/pull/257#discussion_r1118751601) 😛